src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -905,7 +1047,8 @@ void SettingsLoader::_addUserProfileParent(const winrt::com_ptr<implementation::
     }
 }
 
-void SettingsLoader::_addOrMergeUserColorScheme(const winrt::com_ptr<implementation::ColorScheme>& newScheme)
+// returns true if the scheme was successfully added, otherwise false

```

Comment:
to align with jsoref's new guidelines...


```suggestion
// returns true if the scheme was successfully added; otherwise, false
```

Reply:
or, you can skip the weirdness entirely:


```suggestion
// returns whether the scheme was successfully added
```